test/spec/ol/binary.test.js

```diff
@@ -0,0 +1,21 @@
+goog.provide('ol.test.buffer');
+
+describe('ol.binary.Buffer', function() {
+
+  describe('constructor', function() {
+
+    it('can be constructed without arguments', function() {
+      var instance;
+
+      expect(function() {
+        instance = new ol.binary.Buffer();
+      }).to.not.throwException();
+      expect(instance).to.not.be(undefined);
+      expect(instance).to.be.an(ol.binary.Buffer);
```

Comment:
Wouldn't

`expect(instance).to.be.an(ol.binary.Buffer)`

be sufficient?

(Instead of the three assertions)